server/src/main/java/keywhiz/service/daos/SecretSeriesMapper.java

```diff
@@ -39,6 +39,7 @@ public SecretSeries map(SecretsRecord r) {
     return SecretSeries.of(
         r.getId(),
         r.getName(),
+        null,
         r.getDescription(),
         new ApiDate(r.getCreatedat()),
         r.getCreatedby(),
```

Comment:
This is intentionally null. The change to actually update the database schema is separate, and then there'll be a subsequent change to actually start using the new column. This PR is specifically just to update the public API and supporting model classes.

Reply:
Got it, so in the future, this will be `r.getOwner()`?

Reply:
Yup!